app/helpers/content_helper.rb

```diff
@@ -32,10 +23,6 @@ def indent
     tag.span("&nbsp;".html_safe, class: "ml-10px")
```

Comment:
It's outside the scope, but i've been intending to change this to 
```ruby
tag.span("&nbsp;".html_safe, class: "ml-3")
```
Very minor cosmetic difference, but removes cruft from the SCSS. That class has to go.